core-services/image-mirroring/integr8ly/mapping_integr8ly_operator

```diff
@@ -1,3 +1,2 @@
 registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator quay.io/integreatly/integreatly-operator:master
-registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:master
-registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness-osde2e quay.io/integreatly/integreatly-operator-test-harness:osde2e
```

Comment:
Change this to something other than the osde2e tag e.g. ` registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness-osde2e quay.io/integreatly/integreatly-operator-test-harness:osde2e-master`. This will allow you to keep an image synced to quay.io that is built form the latest master, but not trample on the "osde2e" tag.

Reply:
@mikenairn thank you, I've just updated the name of the mirroring image to osde2e-master